types/mudder/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for mudder 1.0
+// Project: https://github.com/fasiha/mudderjs#readme
+// Definitions by: Patrick Gingras <https://github.com/p7g>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class SymbolTable {
```

Comment:
```suggestion
export as namespace mudder;

export class SymbolTable {
```
https://github.com/fasiha/mudderjs#quickstart
> This loads the mudder object into the browser’s global namespace.

This seems to be UMD package, adding `export as namespace ...` fits the bill here.